transformations/oracle/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Oracle Transformation
+permalink: /transformations/oracle/
+---
+
+* TOC
+{:toc}
+
+[Oracle database](https://www.oracle.com/index.html) is a multi-model database management system produced and marketed by Oracle Corporation.
+
+## Example
+After you create a configuration, the first step is to configure database credentials using the `Database Credentials` link:
+
+{: .image-popup}
+![Screenshot - Credentials link](/transformations/oracle/navigate-to-credentials.png)
+
+Fill in the credentials to the database. After testing the credentials, Save them:
```

Comment:
tady by bylo dobry popsat jaky ty credentials maji byt - neco v tomhle smyslu https://help.keboola.com/components/extractors/database/sqldb/#snowflake

asi tam nechces dat master

Reply:
taky bych tu explicitne napsal, ze tohle provozuje vsechny transformace na tebou zadanym stroji a ne na kebolim, narozdil od ostatnich transformaci, cili si musis sam zajistit, aby ti to neco neposralo

Reply:
doplnil jsem to.... uživatele jsem zkusil vytvořit a má přístup jen do svého schématu, tak by to mělo být v pohodě